src/blocks/sound.rs

```diff
@@ -60,6 +73,7 @@
 //! `volume`             | Current volume. Missing if muted. | Number | %
 //! `output_name`        | PulseAudio or ALSA device name    | Text   | -
 //! `output_description` | PulseAudio device description, will fallback to `output_name` if no description is available and will be overwritten by mappings (mappings will still use `output_name`) | Text | -
+//! `active_port`        | Active port, will be `""` if no information available (Same as information in Ports section of `pactl list cards`) | Text | -
```

Comment:
Why `""` instead of just making it absent?

Reply:
You're right, should just make it absent